x/evm/keeper/keeper.go

```diff
@@ -362,7 +362,10 @@ func (k Keeper) getBaseFee(ctx sdk.Context, london bool) *big.Int {
 
 // GetMinGasMultiplier returns the MinGasMultiplier param from the fee market module
 func (k Keeper) GetMinGasMultiplier(ctx sdk.Context) sdkmath.LegacyDec {
-	fmkParmas := k.feeMarketKeeper.GetParams(ctx)
+	fmkParmas, err := k.feeMarketKeeper.GetParams(ctx)
+	if err != nil {
+		panic(err)
```

Comment:
This should be error handling, not panic.

Reply:
https://github.com/xpladev/ethermint/pull/14/commits/7d4ec44e9fbe4243034d52548670ea13eb597878